### DISCUSSION TITLE
List of `%` keywords for custom log format ?

### DISCUSSION THREAD

**Discussion by KaKi87** [allinurl/goaccess#2774] [Q&A] [answered]
Hi,

Where's the list of `%` keywords that can be passed to `--log-format` to parse values in a custom format ?

Also, if said log contains values that GoAccess doesn't support, is there an "ignore" keyword ?

Thanks

**Comment by allinurl** [comment, marked as answer]
I'm not entirely sure what you mean by "list of keywords," but I assume you're referring to the specifiers for a custom format. You can find them in the [[man page](https://goaccess.io/man#custom-log)](https://goaccess.io/man#custom-log). Let me know.

**Reply by KaKi87** [reply]
Yes, thank you !
